scripts/create-template.js

```diff
@@ -1,27 +1,26 @@
-import fs from 'fs';
-import path from 'path';
+import fs from "fs";
+import { createTaskFolder, createTaskFoldersFromRange } from "./task-folder.js";
 
-const today = new Date();
-const year = today.getFullYear();
-const month = String(today.getMonth() + 1).padStart(2, '0');
-const day = String(today.getDate()).padStart(2, '0');
-const folderName = `${year}-${month}-${day}`;
+const arg = process.argv[2];
 
-const folderPath = path.join('tasks', folderName);
+if (!fs.existsSync("tasks")) {
+  fs.mkdirSync("tasks");
+}
 
-if (!fs.existsSync('tasks')) {
-    fs.mkdirSync('tasks');
+if (!arg) {
+  createTaskFolder(new Date());
```

Comment:
As a follow-up, would be nice to reuse `createTaskFoldersFromRange` with just one day. I do understand it requires more work, so only FYI, not a blocker.